src/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -6,39 +6,45 @@ import useMergeRefs from "../../hooks/useMergeRefs";
 import { backwardCompatibilityForProperties } from "../../helpers/backwardCompatibilityForProperties";
 import { baseClassName } from "./ButtonGroupConstants";
 import { ButtonWrapper } from "./ButtonWrapper";
-import "./ButtonGroup.scss";
 import VibeComponentProps from "../../types/VibeComponentProps";
-import { BASE_SIZES, SIZES } from "../../constants/sizes";
+import { BASE_SIZES, SIZES, DialogPosition } from "../../constants";
 import { ButtonType, Size } from "../Button/ButtonConstants";
 import { IconType } from "../Icon/IconConstants";
+import "./ButtonGroup.scss";
 
 type ButtonGroupOption = {
   icon?: IconType;
   leftIcon?: IconType;
   ariaLabel?: string;
   subText?: string;
-  value: number;
+  value: string | number;
```

Comment:
nitpick - consider extracting to a type, or use an existing one (since I see you're using this in a few places)